x11-apps/radeontop/radeontop-1.0.ebuild

```diff
@@ -14,18 +14,19 @@ KEYWORDS="~amd64 ~x86"
 IUSE="nls"
 
 RDEPEND="
-	sys-libs/ncurses
-	x11-libs/libpciaccess
+	sys-libs/ncurses:0=
 	x11-libs/libdrm
-	nls? ( sys-libs/ncurses[unicode] virtual/libintl )
+	x11-libs/libpciaccess
+	x11-libs/libxcb
+	nls? ( sys-libs/ncurses:0=[unicode] virtual/libintl )
 "
 DEPEND="${RDEPEND}
 	virtual/pkgconfig
 	nls? ( sys-devel/gettext )
 "
 
 src_prepare() {
-	epatch_user
+	default
 
 cat > include/version.h << EOF || die
```

Comment:
Should I handle the following QA issue? It seems very minor one.
 * QA Notice: The following shared libraries lack a SONAME
 * /usr/lib/libradeontop_xcb.so